.github/dependabot.yml

```
@@ -11,3 +12,31 @@ updates:
       prefix: "build(deps):"
       prefix-development: "build(deps-dev):"
     versioning-strategy: "increase"
+    groups:
+      dev-deps:
+        dependency-type: "development"
+        applies-to: "version-updates"
+        update-types:
+        - "minor"
+        - "patch"
+    groups:
+      deps:
+        dependency-type: "production"
+        applies-to: "version-updates"
+        update-types:
+        - "patch"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "weekly"
      day: "sunday"
```

Let's return to the previous interval we had